app/i18n/cz/sub.php

```diff
@@ -2,13 +2,13 @@
 
 return array(
 	'api' => array(
-		'documentation' => 'Copy the following URL to use it within an external tool.',// TODO
-		'title' => 'API',// TODO
+		'documentation' => 'Copy the following URL to use it within an external tool.', // TODO
```

Comment:
If possible, please avoid changes in parts of the code not related to the PR. (It was probably automatic, but you might have a way of avoiding it, don't you?)

Reply:
Sure

Reply:
Done